packages/generator-helper/src/dmmf.ts

```diff
@@ -116,6 +115,10 @@ export namespace DMMF {
       model?: SchemaEnum[]
       prisma: SchemaEnum[]
     }
+    fieldRefTypes: {
+      // model?: FieldRefType[]
+      prisma?: FieldRefType[]
```

Comment:
Question out of curiosity: Is this treated as an InputObjectType in the query.ts.